spec/requests/users_request_spec.rb

```diff
@@ -1,12 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe 'UsersController', type: :request do
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job for fix the changes required by the previous reviewer as all tests of this **Project** are now passing. However, I think it would be better if you fix other tests of your previous projects as some of them are not passing. This will make your project to be more professional.
![Screenshot from 2023-03-22 14-51-47](https://user-images.githubusercontent.com/83042742/227007783-f3a27523-335b-45c0-b157-7fde747b7bdc.png)

Reply:
Hello @tresorsawasawa  Thank you for your feedback. I updated the issues accordingly. 
<img width="741" alt="image" src="https://user-images.githubusercontent.com/35809268/227015370-f29aa7f1-9b42-4705-b5b1-1b510e05be2c.png">